docs/source/ruletypes.rst

```diff
@@ -629,6 +647,16 @@ This value will be used if ``generate_kibana_discover_url`` is true and ``kibana
 
 (Optional, string, default ``http://<es_host>:<es_port>/_plugin/kibana/``)
 
+opensearch_url
+^^^^^^^^^^^^^^
+
+``opensearch_url``: The base url of the opensearch application. If not specified, a URL will be constructed using ``es_host``
+and ``es_port``.
+
+This value will be used if ``generate_opensearch_discover_url`` is true and ``opensearch_discover_app_url`` is a relative path
+
+(Optional, string, default ``http://<es_host>:<es_port>/_plugin/kibana/``)
```

Comment:
kibana is mentioned in the URL. Is it correct?

Reply:
Opensearch and ELK share a lot in common and yes you this kibana references still exist